velox/docs/functions/spark/datetime.rst

```diff
@@ -85,6 +85,10 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: next_day(date, string) -> date
+
+    Returns the first date which is later than startDate and named as dayOfWeek.
```

Comment:
Make startDate/dayOfWeek wrapped by \``.
Please also document the behavior for illegal inputs.

Reply:
It would be better if there are some examples added.

Reply:
Please also document all valid values for ``day_of_week``, e.g.,

 ``dayOfWeek`` must be one of the following (case insensitive):
    'SU', 'SUN', 'SUNDAY'
    'MO', 'MON', 'MONDAY'
    'TU', 'TUE', 'TUESDAY'
    'WE', 'WED', 'WEDNESDAY'
    'TH', 'THU', 'THURSDAY'
    'FR', 'FRI', 'FRIDAY'
    'SA', 'SAT', 'SATURDAY'